api.planx.uk/server.ts

```diff
@@ -200,4 +197,12 @@ declare global {
       };
     }
   }
+
+  namespace Http {
```

Comment:
In particular is this a hacky solution? I'm not sure if it's a like for like case compared with the `Express` namespace for which we are already using (abusing?) this technique :)

Reply:
I think this is necessary here and probably the right way to go 👍